From 57eca077053ee0f496f6b2a9a6826b7f0d444a01 Mon Sep 17 00:00:00 2001
From: Akarshan Biswas <akarshanbiswas@fedoraproject.org>
Date: Mon, 8 Jul 2019 18:32:46 +0530
Subject: [PATCH] Add toggle for distro version check

---
 lib/config.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/config.js b/lib/config.js
index 58c40af..3196f6c 100755
--- a/lib/config.js
+++ b/lib/config.js
@@ -102,6 +102,7 @@ Config.prototype.buildArgs = function () {
     brave_referrals_api_key: this.braveReferralsApiKey,
     enable_hangout_services_extension: this.enable_hangout_services_extension,
     enable_cdm_host_verification: this.brave_enable_cdm_host_verification,
+    enable_distro_version_check: this.is_official_build
   }
 
   if (process.platform === 'darwin' && this.targetOS !== 'ios') {
-- 
2.21.0

